### PR TITLE
feat(frontend): display the preview video on click on the preview

### PR DESCRIPTION
**Pull request by gbonaspetti**
#5849 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend_app_file/src/component/PreviewComponent.jsx [hunk 1]
```diff
@@ -82,13 +82,24 @@ export class PreviewComponent extends React.Component {
   //   }
   // }
 
+  handleClickPreview = () => {
+    const { props, state } = this
+    if (props.isVideo) {
+      this.props.onTogglePreviewVideo()
+    } else {
+      if (state.jpegPreviewLoadingState === IMG_LOAD_STATE.LOADED && props.isJpegAvailable) {
```
Comment (Skylsmoi) [inline frontend_app_file/src/component/PreviewComponent.jsx:90]: } else if { // ^^

File: frontend_app_file/src/component/FileComponent.jsx [hunk 2]
```diff
@@ -35,9 +35,7 @@ export class FileComponent extends React.Component {
     const videoWrapper = document.createElement('div')
     videoWrapper.id = 'videoWrapperDiv'
     videoWrapper.className = 'file__previewVideo open'
-    videoWrapper.onclick = (e) => {
-      this.props.onClickClosePreviewVideo(e)
-    }
+    videoWrapper.onclick = () => this.props.onTogglePreviewVideo()
```
Comment (Skylsmoi) [inline frontend_app_file/src/component/FileComponent.jsx:38]: this is equivalent to:
`videoWrapper.onclick = this.props.onTogglePreviewVideo`
unless you specifically don't want to pass the event object

File: frontend_app_file/src/component/PreviewComponent.jsx (hunk 1 above)
Comment (gbonaspetti) [reply frontend_app_file/src/component/PreviewComponent.jsx:90]: Fixed

File: frontend_app_file/src/component/FileComponent.jsx (hunk 2 above)
Comment (gbonaspetti) [reply frontend_app_file/src/component/FileComponent.jsx]: Fixed

**Review by Haskaris — CHANGES_REQUESTED** [review]
You just have to change the title of the feature and it's ok
Here you display the preview video on click on the preview, you don't actually play the video

**Review by Haskaris — APPROVED** [review]

**Comment by Haskaris** [comment]
Tested on branch.

**Pull request merged**